core/src/main/java/io/aiven/klaw/service/UtilControllerService.java

```diff
@@ -82,6 +85,17 @@ public class UtilControllerService {
 
   @Autowired private ConfigurableApplicationContext context;
 
+  private Boolean isCoralBuilt;
+
+  @PostConstruct
```

Comment:
its better with InitializingBean and afterPropertiesSet implementation. ex : ManageDatabase